src/screens/Lending/EnterDepositAmount.js

```diff
@@ -113,7 +114,7 @@ const EnterDepositAmount = ({
 }: Props) => {
   const preselectedAssetSymbol: string = navigation.getParam('symbol');
   const [selectedAssetSymbol, setSelectedAssetSymbol] = useState(preselectedAssetSymbol);
-  const [depositAmount, setDepositAmount] = useState(null);
+  const [depositAmount, setDepositAmount] = useState('');
```

Comment:
Just wondering if we can keep it as `null`? Recently we're trying to get rid of positive fallback values (empty array, empty object, empty string).